### DISCUSSION TITLE
how could I get frames from an m3u8 video streaming link ?

### DISCUSSION THREAD

**Discussion by vynxc** [unoplatform/uno#10315] [Q&A]
Basically I would like to get a frame every 10 seconds from a m3u8 video link. If possible I'd like to use the MediaPlayerElement.

**Comment by jeromelaban** [comment]
This particular feature is not necessarily available natively on all platforms. Still, as Uno does not provide this feature, you may need to use underlying native APIs to do this, depending on the platforms you want to target.

**Reply by vynxc** [reply]
Do you know of package that does this? I don't seem to find one
